source/scripts/Utility.js

```diff
@@ -6,3 +6,20 @@ module.exports.queryURL = function(name) {
     var results = new RegExp("[\\?&]" + name + "=([^&#]*)").exec(location.search)
     return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "))
 }
+
+
+module.exports.Point = class Point {
+    constructor(point = {}) {
+        this.x = point.x || 0
+        this.y = point.y || 0
+    }
+    toString() {
+        return this.x + "x" + this.y
+    }
+    toPoint(that = {}) {
```

Comment:
hmm, yeah it would

not sure what a good name would be... maybe `getTranslated`?

yup, had just done a quick search to see if it was used lol